src/selectors/exportContext.ts

```diff
@@ -86,7 +86,9 @@ export const exportContext = (
   // If it is root, then do not convert it to markdown
   const currentThought = head(context)
   const markdownText: string =
-    format === 'text/markdown' && !isHome([currentThought]) ? turndownService.turndown(currentThought) : currentThought
+    format === 'text/markdown' && !isHome([currentThought]) && !currentThought.includes('=')
```

Comment:
You should use `!currentThoughts.startsWith('=')` instead, otherwise it will have false positives like "1 + 1 = 2" that are not metaprogramming attributes.